### PR TITLE
Pylint integration (in progress..)

### LINKED ISSUE #187: Running pylint for Code quality checks
## Description
This issue is to test all the test cases of features in the project.
Add pylint to the configuration in the deploying script, and return an exit code of 0 only if all files exceed the score of 7.0

## Test Cases
Sample output for pylint:
C: 61, 0: Function name "dumpErrors" doesn't conform to snake_case naming style (invalid-name)
C: 61, 0: Missing function docstring (missing-docstring)
C: 67, 0: Missing function docstring (missing-docstring)

------------------------------------------------------------------
Your code has been rated at 3.75/10 (previous run: 5.22/10, -1.47)
The code rating should be > 7.0 for each python script in the repository.

## Definition of Done
- [ ] At least one test case tested.
- [x] Approval by 1 mentor.

## Estimation
10 hours

### PR DESCRIPTION
**Pull request by NidhiKJha**
### Description
Added a bash script to deploy.sh to extract the scores from each python file in the project and emit exit 0 only if all files have score above 7.

Fixes #187

## How to run
Install [https://www.pylint.org/#install](pylint)
.
chmod a+x deploy.sh
./deploy.sh

## Issues
1. find currently only finds the python files in the root directory.
2. Needs better handling for exit code emission.

Your inputs are requested for modifications.

### CODE REVIEW

**Comment by NidhiKJha** [comment]
> This PR is not linked to any issue. Please follow the template and link it to an issue.

Fixes #187

File: deploy.sh [hunk 1]
```diff
@@ -47,3 +47,13 @@ else
     echo "Publishing failed."
     exit 2
 fi
+# Pylint code quality check
+find . -name "*.py" -depth -exec pylint {} \; |   # Finding files with .py extension
+grep -oE "\-?[0-9]+\.[0-9]+" |                    # Extract the score from each message
+awk 'NR==1 || NR % 4 == 0' |                      # Previuos command will generate garbage numbers as weel, this cuts away unnecasary data
```
Comment (agarwalrounak) [inline deploy.sh]: Please make the comment in this line free from spelling errors.

Comment (ramitsawhney27) [reply deploy.sh]: No need for these comments besides the one on line 50.

File: deploy.sh [hunk 2]
```diff
@@ -47,3 +47,13 @@ else
...
+while read -r i
+do  
+    if (( $(echo "$i < 7.00" |bc -l) )); then
+        exit 2
```
Comment (ramitsawhney27) [inline deploy.sh:57]: Why is the exit code 2?

**Pull request closed without merging**